MobSF/forms.py

```diff
@@ -16,14 +16,19 @@ def errors_message(form):
         form.errors.get_json_data() django 2.0 or higher
 
         :return
-        example { "error": "file This field is required." }
+        example
+        {
+        "error": {
+            "file": "This field is required.", 
+            "test": "This field is required."
+            }
+        }
         """
-        errors_messages = []
-        for k, value in form.errors.get_json_data().items():
-            errors_messages.append(
-                (k + ' ' + ' , '.join([i['message'] for i in value])))
-        return '; '.join(errors_messages)
+        data = form.errors.get_json_data()
+        for k, v in data.items():
+            data[k] = '; '.join([value_detail['message'] for value_detail in v])

```

Comment:
`'; '` ?

Reply:
Will that affect this format?
```
   {
        "error": {
            "file": "This field is required.", 
            "test": "This field is required."
            }
        }
```

Reply:
No, because the origin data is 
```json
{
  "error": {
    "file": [
      {
        "message": "This field is required.",
        "code": "required"
      }
    ],
    "test": [
      {
        "message": "This field is required.",
        "code": "required"
      }
    ]
  }
}
```

the value of file and test is list, that means there may be multiple values